task_12/gulpfile.js

```diff
@@ -0,0 +1,46 @@
+var gulp = require('gulp');
+var sass = require('gulp-sass');
+var cleanCSS = require('gulp-clean-css');
+var imagemin = require('gulp-imagemin');
+var concat = require('gulp-concat');
+var uglify = require('gulp-uglify');
+const autoprefixer = require('gulp-autoprefixer');
```

Comment:
Make code consistent. All modules except this are defined as `var`. Why?

Reply:
Because of stupidity and inattention
Corrected